usaspending_api/awards/models/transaction_fabs.py

```diff
@@ -89,7 +89,7 @@ class TransactionFABS(models.Model):
     ultimate_parent_unique_ide = models.TextField(blank=True, null=True)
     uri = models.TextField(blank=True, null=True, db_index=True)
     submission_id = models.IntegerField(blank=True, null=True)
-    unique_award_key = models.TextField(null=True, db_index=True)  # From broker.
```

Comment:
Obviously the field is from Broker. The comment was useful during the transition from USAspending-derived unique keys to Broker-derived unique keys. It is no longer useful